knowledge-base/how-to-split-a-nested-tabel-on-two-pages.md

```diff
@@ -0,0 +1,42 @@
+---
+title: How to split a nested table on two pages
+description: A common requirement when using nested tables in the report is to force them to break between pages. 
+type: how-to
+page_title: Render nested tables on multiple pages
+slug: how-to-split-a-nested-tabel-on-two-pages
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td>All</td>
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+A common requirement when using *nested* tables in the report is to force them to break between pages. In this article, we will explain how to achieve that behavior.  
+  
+## Solution
+
+By default, [Table/CrossTab/List item](../table-working-with-table-cross-table-list-items) is kept together when the report is paged.  
```

Comment:
This should be in the description. For example:
"By default, [Table/CrossTab/List item](../table-working-with-table-cross-table-list-items) is kept together when the report is paged. However, a common requirement when using *nested* tables in the report is to force them to break between pages. In this article, we will explain how to achieve that behavior."